connector/connect/src/test/scala/org/apache/spark/sql/connect/planner/SparkConnectProtoSuite.scala

```diff
@@ -267,6 +267,12 @@ class SparkConnectProtoSuite extends PlanTest with SparkConnectPlanTest {
       sparkTestRelation.summary("count", "mean", "stddev"))
   }
 
+  test("Test crosstab") {
+    comparePlans(
+      connectTestRelation.crosstab("id", "name"),
+      sparkTestRelation.stat.crosstab("id", "name"))
```

Comment:
Can we also add the `stat` layer to the DSL so DSL can have the same invoke chain: relation.stat.xxx?